packages/kit/src/core/build/index.js

```diff
@@ -323,13 +322,13 @@ async function build_server(
 					amp: ${config.kit.amp},
 					dev: false,
 					entry: {
-						file: ${s(prefix + client_manifest[client_entry_file].file)},
-						css: ${s(Array.from(entry_css).map(dep => prefix + dep))},
-						js: ${s(Array.from(entry_js).map(dep => prefix + dep))}
+						file: assets + ${s(prefix + client_manifest[client_entry_file].file)},
```

Comment:
`assets` is not in quotes on this line, but is on the next line. is that on purpose?

Reply:
yep! we're just creating expressions that will resolve to the runtime (not prebaked) value of `assets`